bin/harbor-restore.sh

```diff
@@ -165,7 +165,10 @@ log_info "Cleaning up restore artifacts..."
 "${root_path}/bin/ck8s" ops kubectl sc delete -n harbor -f "${TMP_JOB_FILE}"
 "${root_path}/bin/ck8s" ops kubectl sc delete configmap -n harbor restore-harbor
 
+# Reinitialize
+"${root_path}/bin/ck8s" ops kubectl sc delete job -n harbor init-harbor-job
+"${root_path}/bin/ck8s" ops helmfile sc sync -l app=harbor
+
```

Comment:
Maybe add a wait here to make sure the init job runs succesfully?

Reply:
Good idea

Reply:
https://github.com/elastisys/compliantkubernetes-apps/pull/2681/commits/ea6ea7114fe1026670195b60fb19c7777897d2c4